src/main/java/no/nav/veilarbvedtaksstotte/service/VedtakService.java

```diff
@@ -286,10 +287,13 @@ void validerUtkastForUtsending(Vedtak vedtak, Vedtak gjeldendeVedtak) {
             throw new IllegalStateException("Vedtak mangler innsatsgruppe");
         }
 
-        if (skalHaBeslutter(innsatsgruppe) || vedtak.isBeslutterProsessStartet()) {
+        boolean isBeslutterProsessStartet = vedtak.getBeslutterProsessStatus() == KLAR_TIL_BESLUTTER;
```

Comment:
Samme problem som tidligere, kan vurdere å lage et par metoder i VedtakUtils for å sjekke om beslutter prosess er startet osv